repositories.yaml

```diff
@@ -9,6 +32,12 @@ repositories:
     pretty_string: Stackable Commons
     product_string: commons
     url: stackabletech/commons-operator.git
+    resources:
+      cpu_limit: 200m
+      memory_limit: 1024Gi
+      cpu_request: 200m
+      memory_request: 1024Gi
```

Comment:
Not a huge fan of maintaining these overrides _here_, but I can see why it's done that way…

Reply:
Me neither, but I thought it was preferrable to doing it inline in the templates and couldn't come up with a better way.

Reply:
Actually, can't the operators put it in their values.yaml? That way it would override the templated values but still be overridden in turn by any user-provided config.

Reply:
They sort of already do:

https://github.com/stackabletech/nifi-operator/blob/8f1797e1137207547bdcbddc38c122e48b50c0e2/deploy/helm/nifi-operator/values.yaml#L34

My intention was to keep this as is, to avoid changing everything but have a default in place that is enforced if the user doesn't set anything.

But maybe I misunderstood your intention.